ci/build_container/build_recipes/zlib.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+VERSION=1.2.11
+
+wget -O zlib-$VERSION.tar.gz https://github.com/madler/zlib/archive/v$VERSION.tar.gz
+tar xf zlib-$VERSION.tar.gz
+cd zlib-$VERSION
+./configure
+make
```

Comment:
Can we use make install here? (Kind of like https://github.com/lyft/envoy/blob/master/ci/build_container/build_recipes/cares.sh). (Also specify V=1 if possible so that we can check compiler options on the build).